exercise-core/src/main/java/com/examples/attsw/exercise/core/controller/EmployeeController.java

```diff
@@ -20,7 +20,7 @@ public String getAllEmployees() {
 	}
 
 	public String getEmployeeById(String id) {
-		return employeeService.getEmployeeById(id).getName();
+		return employeeService.getEmployeeById(id).toString();
```

Comment:
for didactic and exercise purposes, it would be nice to open a new issue about this bug in EmployeeController and a separate PR, that only fixes the problem in EmployeeController (and updates the corresponding test).

In this PR you can simply merge with the branch of the other PR even if it's not yet approved (or cherry pick the corresponding commits).

No merge conflicts would arise when both PR are merged.

Reply:
I opened #23 and correctly solved the bug in a new branch, so i opened a new PR on my fork with a 'Closes #23' comment but it didn't work on merging (maybe i had to open this issue on my fork). However i manually closed the problem.